experimental/adaptive-tool/package.json

```diff
@@ -3,9 +3,9 @@
     "author": "Microsoft Corp.",
     "displayName": "Bot Framework Adaptive Tools",
     "description": "Bot Framework Adaptive Tools",
-    "version": "0.1.0-preview",
+    "version": "0.1.0",
     "license": "MIT",
-    "publisher": "Microsoft",
+    "publisher": "BotBuilder",
```

Comment:
>BotBuilder [](start = 18, length = 10)

Is that how we publish our other packages?

Reply:
Due to the way our account is configured in the Visual Studio Marketplace, this is required.